RetailCoder.VBE/UI/SourceControl/SourceControlBindings.cs

```diff
@@ -34,15 +30,13 @@ public override void Load()
             Bind<IDeleteBranchView>().To<DeleteBranchForm>();
             Bind<IMergeView>().To<MergeForm>();
 
-            //presenters
-            Bind<IChangesPresenter>().To<ChangesPresenter>();
-            Bind<IBranchesPresenter>().To<BranchesPresenter>();
-            Bind<ISettingsPresenter>().To<SettingsPresenter>();
-            Bind<IUnsyncedCommitsPresenter>().To<UnsyncedCommitsPresenter>();
+            //factories 
+            // todo: check on note below
+            // ninject is complaining about also having a SourceControlProviderFactoryProxy and a FolderBrowserFactoryProxy
```

Comment:
Wait, no - the `assemblies` enumerable doesn't contain the `Rubberduck.SourceControl` assembly... not sure what's going on here.